packages/@aws-cdk/aws-ecr/test/test.repository.ts

```diff
@@ -377,7 +377,12 @@ export = {
             ],
             "Effect": "Allow",
             "Principal": "*",
-            "Resource": "*",
+            "Resource": {
+              "Fn::GetAtt": [
+                "TestHarnessRepoAA7E9724",
```

Comment:
This can't work though. Will lead to a dependency cycle, as this is the `GetAtt` of the repository that is currently being created. Try `resourceSelfArns: []`.

Reply:
That's a good point, I'll look into a proper fix.